pkg/catalog/configmeta.go

```diff
@@ -116,6 +136,101 @@ func servicesForRole(
 	return result
 }
 
+// genconfigConnections will look at the cluster spec
+// and generates a map of configmap type and corresponding
+// configmaps to be connected to the given cluster
+func genconfigConnections(
+	cr *kdv1.KubeDirectorCluster,
+) (map[string]map[string]map[string]string, error) {
+
+	models := make(map[string]map[string]string)
```

Comment:
Let's not use the term model in our var names, since these data structures are more generic than that.